spec/services/api/create_params_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Api::CreateParams do
+  describe "#permissions" do
+    it "allows scalar model attributes" do
+      pl_permissions = Api::CreateParams.new(Api::Model.new('plant_line'), {}).permissions
+      # FIXME plant_variety_name - shouldn't it be removed?
```

Comment:
Ah, all right, I though this is about PV. Well, I guess this is one of several "leave it for later curation" cases - perhaps some PLs couldn't be mapped to PVs so Wiktor decided it would be safe to leave that attribute.

So removing that from the DB is rather out of question (or am I wrong - @nowakowski Piotr?). In such a case I vote we shouldn't hide it in API either, though it looks kind of strange.